bin/release.sh

```diff
@@ -1,39 +1,55 @@
 #!/bin/sh
 set -e
 
-# echo  "Checking that you can publish to npm..."
-
-# at some point we should create a team and check if user exists in a team
-# ! npm team ls developers | grep -q $NPM_USER
-
-NPM_USER=$(npm whoami)
-if ! [ "govuk-patterns-and-tools" == "$NPM_USER" ]; then
-  echo "⚠️ FAILURE: You are not logged in with the correct user."
-  exit 1
-fi
-
-echo "📦  Publishing package..."
-
-# Try publishing
-cd package
-npm publish
-echo "🗒 Package published!"
-cd ..
-
-# Extract tag version from ./package/package.json
-ALL_PACKAGE_VERSION=$(node -p "require('./package/package.json').version")
-TAG="v$ALL_PACKAGE_VERSION"
-
-if [ $(git tag -l "$TAG") ]; then
-    echo "⚠️ Tag $TAG already exists"
-    exit 1
+echo "Starting a release..."
+echo " "
+echo "This will:"
+echo "- check that you're logged in to npm as the correct user"
+echo "- publish the package if it has not been published already"
+echo "- create a new tag if the current git tag does not match the latest published tag"
```

Comment:
This implies that if the tag already exists this will just do nothing, whereas from my reading it looks like it'll exit out with an error?